.travis.yml

```diff
@@ -78,7 +78,8 @@ before_install:
 
 after_success:
  - if [ "$TASK" = "coverity" ]; then cat /home/travis/build/${TRAVIS_REPO_SLUG}/tests/cov-int/scm_log.txt; fi
- - if [ "$TASK" = "coverage" ]; then cd tests && coveralls --gcov /usr/bin/gcov-5 -b . -e include -e harmony -e mocks -e config.h -e system_config -E '.*Test\.cpp$' --gcov-options '\-lp' --follow-symlinks > /dev/null; fi
+ - if [ "$TASK" = "coverage" ]; then cd tests && coveralls --gcov
+   /usr/bin/gcov-5 -b . -e include -e harmony -e mocks -e config.h -e system_config -e test --gcov-options '\-lp' --follow-symlinks > /dev/null; fi
```

Comment:
/usr/bin/gcov-5 is the argument to the --gcov flag, so can we leave it attached to that spacing wise please.